pkg/master/master.go

```diff
@@ -230,7 +230,7 @@ func setDefaults(c *Config) {
 		// We should probably allow this for clouds that don't require NodePort to do load-balancing (GCE)
 		// but then that breaks the strict nestedness of ServiceType.
 		// Review post-v1
-		defaultServiceNodePortRange := util.PortRange{Base: 30000, Size: 2767}
+		defaultServiceNodePortRange := util.PortRange{Base: 30000, Size: 2768}
```

Comment:
I was actually going off the error message printed in #9318:

```
 Node port range unspecified. Defaulting to 30000-32766.
```

30000-30000 have size 1, because they're both inclusive.  So I _think_ that Size=2768 includes 32767 but not 32768.

I'll add a unit test or something so we don't have to spend brain cycles thinking about it :-)  (It's too bad golang doesn't have asserts!)